test/eslint-test.js

```diff
@@ -161,7 +161,7 @@ describe('broccoli-lint-eslint', function() {
       yield output.build();
 
       let result = output.read();
-      expect(Object.keys(result)).to.deep.equal(['.eslintrc.lint-test.js', 'a.lint-test.js', 'b.lint-test.js']);
+      expect(Object.keys(result)).to.deep.equal(['.eslintrc.js', 'a.lint-test.js', 'b.lint-test.js']);
```

Comment:
why the file name change?

Reply:
So ESLint made a change (See https://github.com/eslint/eslint/issues/10341) where dotfiles became ignored by ESLint. As a result, the file does not get changed by the filter.

As this change is breaking anyway, and the only to fix is to update the .eslintignore file (issue here: https://github.com/ember-cli/ember-cli/issues/8070), I just edited the tests to make them follow what ESLint 5+ does.